### DISCUSSION TITLE
Sorting songs by year and track number

### DISCUSSION THREAD

**Discussion by akrogun** [fast4x/RiMusic#487] [Ideas]
Just a thought...
I notice the database have tables like Album, SongAlbumMap, etc.
And in table Album has "year" row, and in SongAlbumMap has "position" row. I believe this "position" row are the track number.

I would be appreciate if next update the application can sort a custom playlist by the year and by the track number. Put that sorting behaviour in the Setting - UI or somewhere else.

Thank you.

**Comment by ikanakova** [comment]
@akrogun I think you should open a feature request in issue for this. I like the idea.

**Reply by akrogun** [reply]
Done. I put this issue on Feature request [#488](https://github.com/fast4x/RiMusic/issues/488#issue-2098367696)

**Comment by fast4x** [comment]
#488 
Implemented
